src/styles/blocks/cell.scss

```diff
@@ -0,0 +1,30 @@
+.cell {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  @include square;
+
+  background-color: #eee;
+  border: 1px solid black;
+
+  transition: 0.5s background-color ease, 0.5s transform ease;
+
+  &:hover {
+    background-color: pink;
+
+    cursor: pointer;
+
+    transform: translateY(-20px);
+  }
+
+  @for $index from 1 through 31 {
+    &:nth-child(#{$index})::before {
+      content: "#{$index}";
+
+      font-family: Arial, Helvetica, sans-serif;
```

Comment:
Are u sure, we need font-styles in the loops? Try to add it to the cell.

Reply:
Good point, fixed